lxd/ip/link.go

```diff
@@ -152,3 +305,75 @@ func (l *Link) Delete() error {
 	}
 	return nil
 }
+
+// BridgeVlanAdd adds a new vlan filter entry
+func (l *Link) BridgeVlanAdd(vid string, pvid, untagged, self, master bool) error {
+	cmd := []string{"vlan", "add", "dev", l.Name, "vid", vid}
+	if pvid == true {
+		cmd = append(cmd, "pvid")
+	}
+	if untagged == true {
+		cmd = append(cmd, "untagged")
+	}
+	if self == true {
+		cmd = append(cmd, "self")
+	}
+	if master == true {
+		cmd = append(cmd, "master")
+	}
+
+	_, err := shared.RunCommand("bridge", cmd...)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+// BridgeVlanDelete removes an existing vlan filter entry
+func (l *Link) BridgeVlanDelete(vid string, pvid, untagged, self, master bool) error {
```

Comment:
@presztak according to the manpage of the `bridge` command:

```
   bridge vlan delete - delete a vlan filter entry
       This command removes an existing vlan filter entry.

       The arguments are the same as with bridge vlan add.  The pvid and un‐
       tagged flags are ignored.
```

The pvid and untagged arguments are ignored, so we could probably get away with not having them as arguments to this function to make it easier on the caller to reason about.